### PR TITLE
Enhance comment with dynamic text alignment

### PR DESCRIPTION
**Pull request by lahmar-amal**
For rtl purposes we need the alignment of text even in the comment to be consistent!

### CODE REVIEW

**Review by tokland — COMMENTED** [review]
Thanks for the PR! Could you please push it without indentation/whitespace changes?

File: lib/Sheet.js [hunk 1]
```diff
@@ -716,6 +716,8 @@ class Sheet {
      * @param {string} [opts.text] - The comment text to set.
      * @param {string} [opts.width] - Comment box width.
      * @param {string} [opts.height] - Comment box height.
+     * @param {string} [opts.horizontalAlignment] - Comment horizontalAlignment.
+     * @param {string} [opts.textAlign] - Comment textAlign.
```
Comment (tokland) [inline lib/Sheet.js:720]: Very minor, but we should probably keep the same order of the new two options in Cell and Sheet.

File: lib/types.d.ts [hunk 2]
```diff
@@ -399,6 +399,8 @@ declare namespace XlsxPopulate {
     text: string;
     width: string;
     height: string;
+    textAlign: string;
+    horizontalAlignment: string
```
Comment (tokland) [inline lib/types.d.ts:403]: To help the user of the library: do you think it makes sense to be more restrictive with the values we accept here, or that would be limiting?

https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.vml.spreadsheet.horizontaltextalignment?view=openxml-3.0.1#remarks

https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#formal_syntax

**Review by tokland — COMMENTED** [review]
Thx, @lahmar-amal! Just some minor comments and we are ready to merge:

File: lib/types.d.ts (hunk 2 above)
Comment (lahmar-amal) [reply lib/types.d.ts]: yes you're absolutely right!

**Review by tokland — APPROVED** [review]
@lahmar-amal , thx!

**Pull request merged**